frontend/src/pages/Profile/Admin/admin.tsx

```diff
@@ -253,19 +292,49 @@ const UsersTableHeader = ({
             <TextInput style={styles.createNew} placeholder={label} />
           </View>
         ))}
+        <TouchableOpacity onPress={() => createUser({ name: '', email: '', role: '', phone: '', address: { street: '', city: '', state: '', zip: '', country: '' } })}>
```

Comment:
hmmmm :thinking_face: ,
These really should be `useState` variables that are updated via the `value` of their respective `<TextInput>`'s. However, the TextInput's are being mapped via the `labels`, @william-beukelman We need
`<TextInput value={name} />`, how could we accomplish this while preserving your label mapping?

Reply:
![image](https://user-images.githubusercontent.com/87157912/235336811-3bb08e01-c187-414b-a02d-266064eca45d.png)